ceph-releases/luminous/ubuntu/16.04/daemon/disk_introspection.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -e
+
+HOSTNAME=$(uname -n)
+MY_NAMESPACE=$(kubectl get pods "$HOSTNAME" -o template --template="{{.metadata.namespace}}")
```

Comment:
This command only queries this pod's current namespace so it's sort of repetitive in that we're saving the current namespace only to specify it in subsequent commands.

I think if we want to query all namespaces to find this pod's namespace use this command instead:
```
MY_NAMESPACE=$(kubectl get pods --all-namespaces -o jsonpath="{.items[?(@.metadata.name == \"${HOSTNAME}\")].metadata.namespace}")
```

Otherwise, we can probably leave out the namespace.